ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gcc_11/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+##########
+# gcc_11 #
+##########
+
+- name: Check if gcc 11.2 is installed on RHEL/CentOS/Ubuntu16
+  shell: /usr/local/gcc11/bin/gcc-11.2 --version 2>&1 > /dev/null
+  ignore_errors: yes
+  register: gcc11_installed
+  when: ansible_distribution == "RedHat" or ansible_distribution == "Fedora" or ansible_distribution == "CentOS" or ansible_distribution == "openSUSE" or (ansible_architecture == "armv7l" and ((ansible_distribution == "Ubuntu" and ansible_distribution_major_version == "16") or ansible_distribution == "Debian"))
+  changed_when: False
+  tags: gcc_11
+
+# Unable to check the checksum of the binary as it'll be different for each architecture's tar.xz file
+- name: Download AdoptOpenJDK gcc-11.2 binary
+  get_url:
+    url: https://ci.adoptopenjdk.net/userContent/gcc/gcc112.{{ ansible_architecture }}.tar.xz
```

Comment:
We currently do not have https://ci.adoptopenjdk.net/userContent/gcc/gcc112.* tarball exist.
if we add "-role: gcc_11" in playbook, it would fail when ansible run(?) 
either we do ignore_failure, or add the tarball to https://ci.adoptopenjdk.net/userContent/gcc/ first.

Reply:
Correct :-)